Streaming/Streaming-To-Youtube.md

```diff
@@ -3,20 +3,29 @@
 We publicly live stream our meetings to YouTube using [Zoom](https://zoom.us/).
 
 ## Prerequisites
-You need the Foundation login credentials for Zoom in order to host the meeting. In addition, your YouTube Account must be a manager of the [Node.js YouTube account](https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ). Ask the TSC Chair for access to Zoom and YouTube.
+
+You need the Foundation login credentials for Zoom in order to host the meeting.
```

Comment:
It says "you need them", but its not clear - when hosting a meeting, do we need to ensure we logout from zoom.us with our normal credentials, and log back in with the foundation ones?

Reply:
Yes you must be logged in with the foundation credential to be able to start the meeting.